cluster/charts/crossplane/templates/default-activation-policy.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if gt (len .Values.provider.defaultActivations) 0 -}}
+apiVersion: apiextensions.crossplane.io/v2alpha1
+kind: ManagedResourceActivationPolicy
```

Comment:
I believe this won't work since the CRD for `ManagedResourceActivationPolicy` wouldn't exist on cluster during helm install. We had other similar cases in the past (e.g. default DeploymentRuntimeConfig), and have a pattern to create this type of resources in the initializer, e.g. https://github.com/crossplane/crossplane/blob/e4e442e27027221fce955c70ed437c192fc22d50/cmd/crossplane/core/init.go#L125

Reply:
aha yes that's probably the root cause of all the e2e failures we're seeing mentioned in https://github.com/crossplane/crossplane/pull/6639/#pullrequestreview-3038923275

Reply:
ah, I forgot that crossplane installs in a non-standard way. 😢

Reply:
added a init step